dino/src/game/score/resources.rs

```diff
@@ -1,6 +1,21 @@
-use bevy::prelude::Resource;
+use bevy::prelude::{Resource, Timer, TimerMode};
```

Comment:
Acho que se a gente for usar mais de uma coisa do bevy, já compensa fazer o star import